docs/docsource/operation/Zenko_CLI/Object_Lifecycle_Management.rst

```diff
@@ -21,36 +20,35 @@ deployment configuration files.
 Operation
 ---------
 
-Lifecycle management conforms partially to the S3 lifecycle management
-syntax described at
+Lifecycle management conforms partially to the S3 lifecycle management syntax
+described at
 https://docs.aws.amazon.com/AmazonS3/latest/dev/object-lifecycle-mgmt.html.
-Zenko lifecycle actions are constrained to expiration actions, and not
-transition actions. Files that exceed a preconfigured temporal threshold
-(for example, 90 days) are expired and deleted from the bucket in which
-they are stored.
+Zenko lifecycle management supports expiration and transition actions. Files
+that exceed a preconfigured temporal threshold (for example, 90 days) are
+expired and deleted from the bucket in which they are stored or transitioned and
```

Comment:
not sure if grammatically correct, but having an extra comma `expired and deleted from the bucket in which they are stored, or transitioned and moved` makes the sentence less verbose, and (for me personally) easier to understand in one reading.